public/javascript/controllers/admin-ctrl.js

```diff
@@ -64,6 +75,9 @@ angular.module('harvestv2')
                 $scope.appLogsSearch  = $scope.appLogs = LogsFactory.query({lo_log_level: 'app_activity'});
             }
 
+            /*
+            This function filters the table of application logs based on the text in the search box - which is passed in the searchText variable
+             */
             $scope.filterAppLogs = function (searchText) {
```

Comment:
@nick22891 as you merge in this pull request. Can you create a issue to address the comment that @matjames007 has raised about moving the filtering to the API rather than the front end.

Otherwise. :rocket: the PR!